proto/pdpb.proto

```diff
@@ -441,6 +446,14 @@ message StoreStats {
     uint64 keys_read = 14;
     // Actually reported time interval
     TimeInterval interval = 15;
+    // Threads' CPU usages in the store
+    repeated RecordPair cpu_usages = 16;
+    // Threads' read disk I/O rates in the store
+    repeated RecordPair read_io_rates = 17;
+    // Threads' write disk I/O rates in the store
+    repeated RecordPair write_io_rates = 18;
+    // Operations' latencies in the store
```

Comment:
Do we need to temporarily remove this? Once we have the corresponding PR for it, we can add it back.